lib/install/validate/validator.go

```diff
@@ -361,8 +363,36 @@ func (v *Validator) compatibility(ctx context.Context, conf *config.VirtualConta
 		return
 	}
 
+	// check session's datastore(s) exist
 	_, err := v.Session.Datastore.AttachedClusterHosts(v.Context, v.Session.Cluster)
 	v.NoteIssue(err)
+
+	// closure to look up a datastore based on a URL from the config & check to see if it's writeable
+	checkDS := func(u *url.URL) {
```

Comment:
Looks like this func is used by all datastore hosts checking, it'll be clearer to have one named method.